catalog/internal/catalog/testdata/test-catalog-sources.yaml

```
@@ -21,4 +21,13 @@ catalogs:
     enabled: true
     properties:
       models:
-      - repository: rhelai1/modelcar-granite-7b-starter
\ No newline at end of file
+      - rhelai1/modelcar-granite-7b-starter
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)